topdown/topdown_test.go

```diff
@@ -279,6 +279,61 @@ p[x] { data.a[i] = x }`,
 	}
 }
 
+// astStore returns a fixed ast.Value for Read.
+type astStore struct {
+	storage.WritesNotSupported
+	storage.TriggersNotSupported
+	storage.PolicyNotSupported
+	path  string
+	value ast.Value
+}
+
+func (a *astStore) NewTransaction(context.Context, ...storage.TransactionParams) (storage.Transaction, error) {
```

Comment:
We could drop the receiver var, but it doesn't matter.
```suggestion
func (*astStore) NewTransaction(context.Context, ...storage.TransactionParams) (storage.Transaction, error) {
```

Reply:
Sure, fixed.

Reply:
Still around? Heh. I'll merge this when green.